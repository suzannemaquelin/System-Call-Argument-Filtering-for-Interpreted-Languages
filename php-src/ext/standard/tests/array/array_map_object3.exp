*** Testing array_map() : class methods as callback function ***
-- accessing parent method from child class --
array(3) {
  [0]=>
  int(1)
  [1]=>
  int(5)
  [2]=>
  int(7)
}
-- accessing child method from parent class --

Warning: array_map() expects parameter 1 to be a valid callback, class 'ParentClass' does not have a method 'staticChild' in %s on line %d
NULL
-- accessing parent method using child class object --
array(3) {
  [0]=>
  int(1)
  [1]=>
  int(5)
  [2]=>
  int(7)
}
-- accessing child method using parent class object --

Warning: array_map() expects parameter 1 to be a valid callback, class 'ParentClass' does not have a method 'staticChild' in %s on line %d
NULL
Done