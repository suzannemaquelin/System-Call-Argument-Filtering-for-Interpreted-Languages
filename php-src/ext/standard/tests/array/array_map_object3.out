Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
*** Testing array_map() : class methods as callback function ***
-- accessing parent method from child class --
array(3) {
  [0]=>
  int(1)
  [1]=>
  int(5)
  [2]=>
  int(7)
}
-- accessing child method from parent class --

Warning: array_map() expects parameter 1 to be a valid callback, class 'ParentClass' does not have a method 'staticChild' in /root/php-src/ext/standard/tests/array/array_map_object3.php on line 51
NULL
-- accessing parent method using child class object --
array(3) {
  [0]=>
  int(1)
  [1]=>
  int(5)
  [2]=>
  int(7)
}
-- accessing child method using parent class object --

Warning: array_map() expects parameter 1 to be a valid callback, class 'ParentClass' does not have a method 'staticChild' in /root/php-src/ext/standard/tests/array/array_map_object3.php on line 57
NULL
Done